src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2703,17 +2703,24 @@ static void searchContextProviders
   std::string                        err;
```

Comment:
Is there any change in this file appart from LM_T LmtForward adddition? It seems so but I would like to have a confirmation on your side, pls.

Reply:
Seems to be only LM_T in this file, yes

Reply:
NTC